expression/function_traits.go

```diff
@@ -226,13 +226,9 @@ var mutableEffectsFunctions = map[string]struct{}{
 	ast.AnyValue:    {},
 }
 
-// some functions like "get_lock" and "release_lock" currently do NOT have
-// right implementations, but may have noop ones(like with any inputs, always return 1)
+// some functions do NOT have right implementations, but may have noop ones(like with any inputs, always return 1)
 // if apps really need these "funcs" to run, we offer sys var(tidb_enable_noop_functions) to enable noop usage
-var noopFuncs = map[string]struct{}{
-	ast.GetLock:     {},
-	ast.ReleaseLock: {},
-}
+var noopFuncs = map[string]struct{}{}
```

Comment:
Are there any other noop functions that may use this in the future? Or should we remove this?

Reply:
Yes, I remember when I wrote this a couple of weeks ago that there was a function that we should add to this list. But I can't remember it right now. We can remove it in a future refactor if that's not the case.